src/components/searchPanel/searchPanel.css

```diff
@@ -0,0 +1,30 @@
+.search-panel {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    padding: 0 12px;
+    max-width: 1073px;
+    width: 100%;
+    height: 54px;
```

Comment:
styles that determine the size should not be in the block, but in the element that is signed like this relative-block__block

Reply:
Do you mean I need to make an extra container with element dimensions?

Reply:
yes. add and add size characteristics inside of it 
relative-block__block{ margin: 0 auto;
    padding: 0 12px;
    max-width: 1073px;}